.github/workflows/create-release-pr.yaml

```
@@ -0,0 +1,67 @@
+# This workflow concerns the preparation of the `changeset` PR and keeping it updated by tracking the changes on `main` branch.
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  create-release-pr:
+    name: Create Release PR
+    runs-on: blacksmith-2vcpu-ubuntu-2204
+    steps:
+      - name: Check if a branch is whitelisted and maintained
+        id: branch-check
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
+          WHITELISTED_BRANCHES=("main" "release/2.*.*")
```

```suggestion
          WHITELISTED_BRANCHES=("main" "releases/*")
```
should keep it generic. we might want to release for v1.x as well which is live in production. Don't need to be super restrictive here, and have to maintain this workflow with releases